### PR TITLE
Enroll User

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**

### CODE REVIEW

File: lib/ui/blocs/deeplink/deeplink_bloc.dart [hunk 1]
```diff
@@ -87,10 +90,17 @@ class DeeplinkBloc extends Bloc<DeeplinkEvent, DeeplinkState> {
       final chain = queryParams['chain']!;
       final dao = queryParams['dao']!;
 
+      final String? enrollSecret = queryParams['enroll_secret'];
```
Comment (gguijarro-c-chwy) [inline lib/ui/blocs/deeplink/deeplink_bloc.dart:93]: parse the enroll secret from the link

File: lib/ui/onboarding/usecases/create_account_use_case.dart [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
+      inviteLinkData: input.inviteLinkData,
+      userName: input.userName,
+      image: input.image,
+    );
+
+    if (result.isValue) {
+      if (input.inviteLinkData.enrollSecret?.isNotEmpty == true) {
+        await _inviteService.redeemInvite(
+          user: result.asValue!.value,
+          secret: input.inviteLinkData.enrollSecret!,
```
Comment (gguijarro-c-chwy) [inline lib/ui/onboarding/usecases/create_account_use_case.dart:31]: use the enroll secret from the link

**Comment by n13** [comment]
LGTM!

**Review by n13 — APPROVED** [review]
Looks good

**Pull request merged**